### DISCUSSION TITLE
Workflow confusion

### DISCUSSION THREAD

**Discussion by bogzbonny** [git-bug/git-bug#1363] [Q&A] [answered]
Hey, I'm a little bit confused about how the workflow for gitbug works. In particular I pulled this repository (git-bug/git-bug) I expected to be able to see issues immediately in the master branch, however then I realized I'd need to perform `git bug pull origin`  - however no git bug issues are received at all - Is this repo not using git-bug? To me this is unexpected behaviour. I was able to setup a github bridge and then pull updates from github for the repo, however I thought the updates would be pushed directly into the git remote and that it would be possible to retrieve them without setting up the bridge for a repo that already has been using git-bug such as this repo (I assume). 

A little bit of clarity on workflow specifics would be useful, I also imagine that a few more commands in the `README.md` might be helpful, specific for those who want to access bugs in a repo which is already setup with git bug, maybe under the workflow diagrams. 

Thanks for your time whoever reads this!

**Comment by LokiMidgard** [comment, marked as answer]
Currently git-bug is not using git-bug for issues, #1221 was created for that. And I think I read another comment somewhere why, but can't find it anymore.

**Comment by sudoforge** [comment]
@LokiMidgard is correct -- we are not currently dogfooding `git-bug`, but i plan to rectify this when i have the time to (#1221).

note that as you experienced, there are a few different sources for issues - in the repository itself, with the native workflow (that is, under `refs/heads/bugs`); and on some bridge, e.g. this github repository.

even when #1221 is completed, users without push access will not be able to push to `refs/bugs/*`, so the primary interaction you'd have would be through the bridge (unless you're just pulling issues for archival purposes, in which case fetching with the non-bridge workflow should be fine).

regarding your note about documentation; i have a larger refactor underway to update, clean up, and organize all of the docs that should be ready soon (#1222).

**Comment by sudoforge** [comment]
we (or, at least, i) am now dogfooding git-bug: https://github.com/git-bug/git-bug/issues/1221#issuecomment-2884336629

> i've manually imported all issues and identities and pushed them to the remote.
> 
> ```
> ➜ for n in bugs identities; do echo "${n} on the remote: " $(git ls-remote origin "refs/${n}/*" | wc -l); done
> bugs on the remote:  408
> identities on the remote:  283
> ```

as a personal goal, i am using git-bug as much as possible, but there are still cases where missing features require me to come to github. notably: milestones (but with #1423, maybe that isn't necessary).
